components/dashboard/src/start/StartPage.tsx

```diff
@@ -105,6 +108,24 @@ export function StartPage(props: StartPageProps) {
                 )}
                 {error && <StartError error={error} />}
                 {props.children}
+                {props.showLatestIdeWarning && (
+                    <Alert closable={true} type="warning" className="mt-4 w-96">
```

Comment:
**thought**: I was thinking whether it could help to introduce a softer and lighter version of this warning alert that is also les verbose as discussed earlier. This could still use the alert component without the background or simply some a text element with an icon. What do you think?

| BEFORE | AFTER |
|-|-|
| <img width="1440" alt="Screenshot 2022-03-18 at 2 30 12 PM" src="https://user-images.githubusercontent.com/120486/159904660-229c5271-f5c0-4d20-9cc5-337ffa274084.png"> | <img width="1440" alt="Screenshot 2022-03-18 at 2 29 46 PM" src="https://user-images.githubusercontent.com/120486/159904662-6a5efa48-d06f-4428-ad50-3ec7f9b04b7f.png"> |

Reply:
Yeah, this's better than the one that has a warning background.

Reply:
Same comment as above [[1](https://github.com/gitpod-io/gitpod/pull/8783/files#r838438500)]

Can we swap the language of "latest" for the exact naming? e.g. when JetBrains 

For JetBrains...
> You are using the EAP version of the JetBrains IDE

And for VS Code...
> You are using the Insiders version of the VS Code

Reply:
**fyi**: I opted for suggesting a general term that is also the same for all editor options here to avoid making the alert complex to implement but also hard to understand for users. If we go with the (unstable) suffix in the checbox this could be updated to the following.

> You've opted in for using the latest release (unstable) for the editor. [Change Preferences]()

Thoughts? Maybe with the current design being explicit here could be better but ideally if were better at surfacing the editor option during workspace loading there would be no much need to be explicit about this. 💭

Reply:
If it's easy to get the editor option in there, let's go with what @loujaybee suggested above in https://github.com/gitpod-io/gitpod/pull/8783#discussion_r838447217 or a simpler version to also match the preference option in user settings. Please, ignore the workspace start error on the top of the page. 🤷 

| VS Code Insiders | IntelliJ IDEA EAP |
|-|-|
| <img width="1440" alt="Screenshot 2022-04-05 at 3 03 07 PM" src="https://user-images.githubusercontent.com/120486/161761458-372bf0f7-a028-47f3-bdba-0f823fc5d7f7.png"> | <img width="1440" alt="Screenshot 2022-04-05 at 3 03 45 PM" src="https://user-images.githubusercontent.com/120486/161761467-2855a1b4-f524-43e5-a14e-73fbb190c344.png"> |

Alternatively, we could also drop the yellow background for this case but using a shorter text could make it ok to keep the yellow background. @mustard-mh Let me know if you need help to decide which version to keep.

Reply:
There are some questions:
* We have both browser IDE and DesktopIDE. Which one should we display `EAP for <jet_brains> IDE` or `Insiders for VS Code`?
  * If we use JetBrains one, users like me (only choose but always use code browser, for network does not support me to use jetbrains gateway very smoothly) don't want this alert
  * For VS Code one ...
* `code insiders` is always here but `goland EAP` is not. If this happens, it means we start `insiders` for code browser, `stable` for goland, so what should we display
* What about `code` for browser stable goland for desktop?

For this situation, server chooses(I choose🤪) to use the easiest one: depends on user setting, whether ide option have `latestImage` or not. So  dashboard should not specialify it too.

WDYT? cc @loujaybee @gtsiolis

Reply:
**question**: Do I understand correctly that we do _not_ support GoLand EAP yet? It seems there's an EAP version for GoLand[[1](https://www.jetbrains.com/go/nextversion/)].

**thought**: The idea of this consolidation of options is to simplify options for the user and hide the complex matrix of options behind a simple checkbox.

**suggestion**: I'd think if we don't support GoLand EAP yet, we can safely drop the alert component when GoLand and Latest are selected. In the future, we could also be more specific about this by introducing more details per option in https://github.com/gitpod-io/gitpod/issues/8816.

**suggestion**: Also, in case this is also unclear, since we provide the latest release for VS Code in :a: Browser and :b: Desktop we should probably display the alert when the Latest is also checked for both of these options.

@mustard-mh I could be missing something but please let me know if there's anything still unclear! 🙏

Reply:
I'm still not entirely sure I understand the core issue here. 💫 

> EAP will be out-of-date somehow 

**question**: How is this possible? Do we manually update some links to get the latest release?

> ... code will be insiders but golang will be stable.

**question**:  How is this possible? To help me understand better what's the issue here, there's a case where a user can have the Latest Release (Unstable) selected and they will get the stable version of an editor like GoLand? If yes, is this because we do not regularly update or use the EAP version and eventually the version we're using is the stable one?

> code insiders is always here but goland EAP is not. If this happens, it means we start insiders for code browser, stable for goland, so what should we display

**thought**: I assume the EAP version of the GoLand will always be ahead or equal with the stable version of the GoLand, right? If yes, I'd think it's ok to keep the alert with the latest release warning since this reflects the user preference to use the latest release using the checkbox. 💭